src/consumer/topic.rs

```diff
@@ -60,7 +63,7 @@ impl<T: DeserializeMessage, Exe: Executor> TopicConsumer<T, Exe> {
             options,
             dead_letter_policy,
         } = config.clone();
-        let consumer_id = consumer_id.unwrap_or_else(rand::random);
+        let consumer_id = consumer_id.unwrap_or_else(|| CONSUMER_ID_GENERATOR.fetch_add(1, Ordering::SeqCst));
```

Comment:
I wonder why we allow users to configure this id. We don't do that in the Java client. It's not inherently wrong, but it seems like it might leak the abstraction and could have unintended consequences.

Reply:
I agree that we can change this manner and remove the consumer id from the config struct.